wagtail/admin/tests/pages/test_create_page.py

```diff
@@ -2037,10 +2037,24 @@ def test_sidebar_private(self):
 
         self.assertEqual(response.status_code, 200)
 
-        self.assertContains(response, '<div class="" data-privacy-sidebar-private>')
+        self.assertContains(
+            response,
+            '''class=""
+            data-controller="w-zone"
+            data-action="w-privacy:changed@document->w-zone#switch"
+            data-w-zone-inactive-class="w-hidden"
+            data-w-zone-mode-value="active"
+            data-w-zone-switch-key-value="!isPublic"''',
+        )
```

Comment:
then it will require selector like `data-privacy-sidebar-private` & `data-privacy-sidebar-public` (which makes python test cases simple and cleaner) or else we have to loop through elements to select exact either privet or public div elements.

Can i re-add those selectors in privacy.html? `wagtail\admin\templates\wagtailadmin\shared\side_panels\includes\status\privacy.html`